postgres_exporter/linux/queries_backrest.yml

```diff
@@ -20,6 +20,8 @@ ccp_backrest_last_info:
             )
             SELECT a.config_file
              , a.stanza
+             , a.backup_data->'backrest'->>'version' AS backrest_version
```

Comment:
```suggestion
             , a.backup_data->'backrest'->>'version' AS backrest_server_version
```

Reply:
Has to be changed in other places as well. Will include this in next commit.

Reply:
Since this data comes from the repo output, I set this to `backrest_repo_version` until we get full confirmation of which version this is actually reporting.

`ccp_backrest_system_version_number` will give the version from the installed binary.

Reply:
```
-bash-4.2$ grep -rnE '2\.35|2\.36' *
archive/crunchy/archive.info:3:backrest-version="2.36"
archive/crunchy/archive.info.copy:3:backrest-version="2.36"
backup/crunchy/backup.info:3:backrest-version="2.35"
backup/crunchy/backup.info:6:20220105-193042F={"backrest-format":5,"backrest-version":"2.35","backup-archive-start":"000000010000000000000004","backup-archive-stop":"000000010000000000000004","backup-info-repo-size":3871781,"backup-info-repo-size-delta":3871781,"backup-info-size":32647705,"backup-info-size-delta":32647705,"backup-timestamp-start":1641411042,"backup-timestamp-stop":1641411045,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":false,"option-backup-standby":false,"option-checksum-page":true,"option-compress":true,"option-hardlink":false,"option-online":true}
backup/crunchy/backup.info.copy:3:backrest-version="2.35"
backup/crunchy/backup.info.copy:6:20220105-193042F={"backrest-format":5,"backrest-version":"2.35","backup-archive-start":"000000010000000000000004","backup-archive-stop":"000000010000000000000004","backup-info-repo-size":3871781,"backup-info-repo-size-delta":3871781,"backup-info-size":32647705,"backup-info-size-delta":32647705,"backup-timestamp-start":1641411042,"backup-timestamp-stop":1641411045,"backup-type":"full","db-id":1,"option-archive-check":true,"option-archive-copy":false,"option-backup-standby":false,"option-checksum-page":true,"option-compress":true,"option-hardlink":false,"option-online":true}
backup/crunchy/20220105-193042F/backup.manifest.copy:3:backrest-version="2.35"
backup/crunchy/20220105-193042F/backup.manifest:3:backrest-version="2.35"
-bash-4.2$ pwd
/var/lib/pgbackrest
-bash-4.2$ pgbackrest --stanza=crunchy version
pgBackRest 2.35
```
seems like a given backup can tell you what version was used for the backup, but all other commands return the version currently installed

(i installed 2.36 and took a full first, then downgraded to 2.35 and took another full)

Reply:
Removed `ccp_backrest_system_version_number` metric. Was only able to get that from primary.

For the version, only a single version field is returned from the info command. And found the answer to the question as far as what `info` itself returns. It's the version that ran the backup. So this wouldn't help monitoring if it was upgraded on the system or not. Keeping it for informational purposes tho.
```
[
  {
    "archive": [
      {
        "database": {
          "id": 1,
          "repo-key": 1
        },
        "id": "14-1",
        "max": "000000010000000000000087",
        "min": "00000001000000000000007C"
      }
    ],
    "backup": [
      {
        "archive": {
          "start": "000000010000000000000080",
          "stop": "000000010000000000000080"
        },
        "backrest": {
          "format": 5,
          "version": "2.35"
        },
        "database": {
          "id": 1,
          "repo-key": 1
        },
        "info": {
          "delta": 37258619,
          "repository": {
            "delta": 4679699,
            "size": 4679699
          },
          "size": 37258619
        },
        "label": "20220105-151134F",
        "prior": null,
        "reference": null,
        "timestamp": {
          "start": 1641413494,
          "stop": 1641413504
        },
        "type": "full"
      },
      {
        "archive": {
          "start": "000000010000000000000081",
          "stop": "000000010000000000000081"
        },
        "backrest": {
          "format": 5,
          "version": "2.35"
        },
        "database": {
          "id": 1,
          "repo-key": 1
        },
        "info": {
          "delta": 37261179,
          "repository": {
            "delta": 4679913,
            "size": 4679913
          },
          "size": 37261179
        },
        "label": "20220105-151153F",
        "prior": null,
        "reference": null,
        "timestamp": {
          "start": 1641413513,
          "stop": 1641413522
        },
        "type": "full"
      },
      {
        "archive": {
          "start": "000000010000000000000087",
          "stop": "000000010000000000000087"
        },
        "backrest": {
          "format": 5,
          "version": "2.36"
        },
        "database": {
          "id": 1,
          "repo-key": 1
        },
        "error": false,
        "info": {
          "delta": 37268706,
          "repository": {
            "delta": 4680479,
            "size": 4680479
          },
          "size": 37268706
        },
        "label": "20220105-151634F",
        "prior": null,
        "reference": null,
        "timestamp": {
          "start": 1641413794,
          "stop": 1641413804
        },
        "type": "full"
      }
    ],
    "cipher": "none",
    "db": [
      {
        "id": 1,
        "repo-key": 1,
        "system-id": 7049754983685137000,
        "version": "14"
      }
    ],
    "name": "golf",
    "repo": [
      {
        "cipher": "none",
        "key": 1,
        "status": {
          "code": 0,
          "message": "ok"
        }
      }
    ],
    "status": {
      "code": 0,
      "lock": {
        "backup": {
          "held": false
        }
      },
      "message": "ok"
    }
  }
]```